### ISSUE TITLE
InputMask focus class (ui-inputwrapper-focus) inaccurately applied

### ISSUE THREAD

**Issue by cadin81** [primefaces/primeng#8219]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
https://stackblitz.com/edit/github-eyybbb

**Current behavior**
`<p-inputMask>` is always with the class `ui-inputwrapper-focus`.

**Expected behavior**
The class `ui-inputwrapper-focus` should only be applied when the internal `<input>` element is focused.

**Patch**
I believe that in order to correct the issue the code `'[class.ui-inputwrapper-focus]': 'focus'` on line 48 in file `src/app/components/inputmask/inputmask.ts` should be changed to `'[class.ui-inputwrapper-focus]': 'focused'`.

**Minimal reproduction of the problem with instructions**
Inspect the `<p-inputMask>` element on the browser and observe its class attribute.

**What is the motivation / use case for changing the behavior?**
I want code that depend on the issue, like UltimaNG, to work correctly. Ultima's floating label doesn't work because of this issue.

**Please tell us about your environment:**

* **OS**
Ubuntu 19.04

* **Angular version:**
Angular 7.2.15
Apparently, Angular doesn't play a role on the problem.

* **PrimeNG version:**
PrimeNG 7.1.2
It is still an issue in the latest version.

* **Browser:**
Chrome 77.0.3865.90 64bits
I believe it's reproducible in every browser.
 
* **Language:**
TypeScript 3.1.3

**Comment by ctaylor-osv** [comment]
This is the same issue as identified in issue #8032. I have pull request, #8034, that resolved. So far, I've not received any comments/feedback on either the issue I opened or the pull request, even though it's an obvious bug and super simple fix.

**Comment by cadin81** [comment]
> This is the same issue as identified in issue #8032. I have pull request, #8034, that fixes the problem. So far, I've not received any comments/feedback on either the issue I opened or the pull request, even though it's an obvious bug and super simple fix.

There goes my hope of a quick fix by the PrimeNG team. :-P

**Comment by cadin81** [comment]
I've made a hack... Put the code on the startup of the application and it should work. Didn't get any visible side effects. Not what I wanted, but well, I'm trying... :(

```
    const workaround = InputMask.prototype as any;
    workaround.focus = false;
    workaround.onInputFocusOriginal = workaround.onInputFocus;
    workaround.onInputFocus = function (event) {
      this.focus = true;
      this.onInputFocusOriginal(event);
    };
    workaround.onInputBlurOriginal = workaround.onInputBlur;
    workaround.onInputBlur = function (event) {
      this.focus = false;
      this.onInputBlurOriginal(event);
    };
```

**Comment by ctaylor-osv** [comment]
@yigitfindikli This can be closed since #8032 has been closed.

**Comment by yigitfindikli** [comment]
@ctaylor-osv thank you.
